cypress/integration/smokeTests/sharedTests/auth-tests.ts

```diff
@@ -99,16 +96,15 @@ function registerRemoteSitesTool(repo: string, name: string) {
 
     cy.visit('/my-tools');
     cy.wait('@tokens');
-    cy.wait(2000); // hardcoded 2s wait is least flaky option right now, revisit in future
-    cy.get('#register_tool_button').click();
+    cy.get('#register_tool_button').should('be.visible').should('be.visible').click();
```

Comment:
accidental double be.visible?

Reply:
Reduced visibility: https://github.com/dockstore/dockstore-ui2/pull/1448/commits/bd22ad2d548e2f837c486a0b101aae8c567f1a73